pandas/tests/io/formats/test_to_csv.py

```diff
@@ -3,6 +3,7 @@
 import sys
 
 import pytest
+from mock import patch
 
```

Comment:
The different Python version builds are failing due to the different ways of importing the mock module which is included in unittest. 

I found an ugly way of fixing this, below but will wait for comments first. 


```python
try:
    from unittest.mock import patch
except ImportError:
    from mock import patch

```

Reply:
Can you use monkeypatch from pytest instead?

Reply:
I did not know of that module! I will try that after work tonight, looks straight forward from the docs.

Reply:
Wow, monkeypatch was right there in the first test argument the whole time :sweat_smile:. I rebased and force pushed away the unittest stuff.